src/webui/api/torrentcreatorcontroller.cpp

```diff
@@ -112,6 +113,16 @@ TorrentCreatorController::TorrentCreatorController(BitTorrent::TorrentCreationMa
 {
 }
 
+QStringList parseUrls(const QString &urlsParam)
+{
+    const QStringList encodedUrls = urlsParam.split(u'|');
```

Comment:
```suggestion
    const QStringList encodedUrls = urlsParam.split(u'|', Qt::SkipEmptyParts);
```

Reply:
> This is intended, empty line means new tier.

Then I suppose an appropriate comment in the code is required to prevent unwanted "improvements" in the future.